ixmp4/data/db/base.py

```diff
@@ -237,7 +240,8 @@ def tabulate(
         if self.session.bind is not None:
             with self.engine.connect() as con:
                 return (
-                    pd.read_sql(_exc, con=con).fillna(np.nan).replace([np.nan], [None])
+                    pd.read_sql(_exc, con=con).replace([np.nan], [None])
+                    # pd.read_sql(_exc, con=con).fillna(np.nan).replace([np.nan], [None]) # noqa
```

Comment:
Can this line be removed?

Reply:
The commented line is the old one that I changed due to the pandas update not wanting to call `fillna(np.nan)`. Through trial and error, I found that all `na` values are already `np.nan` in our test suite, so there's no need to call `fillna(np.nan)`. If that's fine, I can of course remove the line. We might however want to keep this in mind or bake the requirement into a test (as meksor said) to ensure this behavior is intentional/known to others down the line. Let's open a new issue for that and simply remove the line for this PR.